src/template-parameters.json

```diff
@@ -7,7 +7,7 @@
     "macrosInSelect": false,
     "selectItems": [
       {
-        "displayValue": "Init",
```

Comment:
this is actually the method name, so I would give it a display value other than its name.
In fact I think we should even remove the capitals and have them camel cased to avoid unnecessary confusion.